app/models/user.rb

```diff
@@ -68,6 +70,10 @@ def password_reset_expired?
     reset_sent_at < Settings.email_expired.hours.ago
   end
 
+  def feed
+    Micropost.where("user_id = ?", id)
```

Comment:
chỗ này em có thể dùng lợi thế của association đc nhé
```suggestion
    microposts
```